src/lib/MiradorViewer.js

```diff
@@ -38,10 +40,14 @@ class MiradorViewer {
   }
 
   /**
-   * Process config into actions
+   * Process config with plugin configs into actions
    */
   processConfig() {
-    this.store.dispatch(importConfig(this.config));
+    this.store.dispatch(
+      importConfig(
+        deepmerge(this.config, getConfigFromPlugins(this.plugins)),
```

Comment:
```suggestion
        deepmerge(getConfigFromPlugins(this.plugins), this.config),
```

I feel like the explicit configuration should always "win" over the plugin-provided config so the application can override the plugins?

Reply:
Totally 💯